### PR TITLE
 Design system overhaul

### PR DESCRIPTION
**Pull request by aguscruiz**
Created new contextual styles to coexist with the extended palette. Both with light/dark modes. Also reorganized text styles, separating the component specific ones to their own "Components" folder.

### CODE REVIEW

**Comment by AlanBreck** [comment]
@petemill and @fallaciousreasoning: I got the Tailwind transforms fixed. Can't say I'm proud, but it's done.

File: web-components/button/button.story.js.old [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import Button from './button.svelte';
```
Comment (petemill) [inline web-components/button/button.story.js.old:1]: @AlanBreck is this .old file a mistake inclusion?

File: web-components/button/test2.css [hunk 2]
```diff
@@ -0,0 +1,3 @@
+body {
```
Comment (petemill) [inline web-components/button/test2.css:1]: ditto for this

**Review by petemill — APPROVED** [review]
@AlanBreck 's and @aguscruiz 's changes look good!

**Comment by petemill** [comment]
I've fixed my mistake test file inclusion

**Review by AlanBreck — APPROVED** [review]

**Pull request merged**